about_us.html

```diff
@@ -82,29 +82,60 @@
                             vertical
                             class="description-bg">
                     <v-carousel-item v-for="(slide, i) in slides"
-                                     :key="i">
+                                        :key="i">
                         <v-sheet height="100%">
-                            <v-row>
-                                <div class="name">
-                                    {{slide.name}}
-                                </div>
-                            </v-row>
-                            <v-row>
-                                <div class="desc-title">
-                                    {{slide.title}}
-                                </div>
-                            </v-row>
-                            <v-row>
-                                <div class="description">
-                                    {{slide.description}}
-                                </div>
-                            </v-row>
-                            <v-row>
-                                <div class="skills">
-                                    <ul v-for="skill in slide.skills">
-                                        <li>{{skill.name}} : {{skill.power_level}}</li>
-                                    </ul>
-                                </div>
+                            <v-row class="description-row">
+                                <v-col class="description-sizer" cols="12" md="6" style="text-align: left;">
+                                    <div class="name">
+                                        {{slide.name}}
+                                    </div>
+                                    <div class="desc-title">
+                                        {{slide.title}}
+                                    </div>
+                                    <div class="desc-title">
+                                        Top Skills:
+                                    </div>
+                                    <div class="skills">
+                                        <ul style="list-style-type: none">
+                                            <li v-for="skill in slide.skills"><b>{{skill.type}}</b>{{skill.skills}}</li>
+                                        </ul>
+                                    </div>
+                                </v-col>
+                                <v-col class="description-sizer" style="text-align: left" cols="12" md="6">
+                                    <div class="desc-links desc-links-before">
+                                        <template v-if="slide.website != ''">
```

Comment:
We can change all these from `v-if="slide.website != ''"` to `v-if="slide.website"` or that equivalent. An empty string evaluates to false so we can just check if there's a string value in our `slide.[link]`